examples/deno-kv-demo/README.md

```
@@ -0,0 +1,134 @@
+
+### Always Close the Store
+
+```typescript
+await adapter.close();
+```
+
+## How Deno KV Storage Works
+
+Each document is stored as a separate KV entry with a structured key:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)